ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,67 @@
+<!--
+
+Hi there! Here's a few pointers to help us help you with your issue as quickly
+as possible.
+
+We prefer that the RSpec Rails issue tracker be mainly used for bug reports.
+Feature requests or general requests for help should go to our google group:
+rspec@googlegroups.com. We use that as our primary location for higher level
+RSpec discussions.
+
+If you are filing a bug report, there's a few steps it'd be super if you could
+follow. If you can't do all of these, don't worry, but if you can it'll help us
+resolve your bug faster!
+
+-->
+
+## Observed behaviour
+
+<!--
+please provide a concise description of the behaviour you are observing with
+RSpec and Rails
+-->
+
+## Expected behaviour
+
+<!--
+Please provide a description of what you expect to be happening, and how that
+differs from the current behaviour.
+-->
+
+## Can you provide an example app?
```

Comment:
you want a link, so you should say link...

Reply:
Isn't mention on line 45?